lua/wikis/commons/MatchPage/Base.lua

```diff
@@ -266,6 +268,46 @@ function BaseMatchPage:renderGame(game)
 	error('BaseMatchPage:renderGame() cannot be called directly and must be overridden.')
 end
 
+---@protected
+---@param match MatchPageMatch
+---@return Widget[]
+function BaseMatchPage:renderMapVeto(match)
+	if not match.extradata or not match.extradata.mapveto then
```

Comment:
should we check for `match.extradata.mapveto` beeing (deep) empty instead of nil?